.gitignore

```diff
@@ -374,3 +374,5 @@ terraform_output.json
 
 # Ignore VS Code ionide extension temp folder
 /.ionide/
+/Deployment/src/APIM/.terraform.lock.hcl
+/Deployment/src/APIM/external-notification-service.openApi.yaml
```

Comment:
Why are we ignoring this?

Reply:
They are just to support local development. Terraform requires open api spec yaml file in it's directory and pipeline copies it there. 
`  api_openapi_spec_path           = file("${path.module}/external-notification-service.openApi.yaml")`

In order to have Terraform working on development machine we keep a copy of spec in APIM folder and updated gitignore to ignore them.